flowise-embed/src/utils/fileClassificationUtils.ts

```diff
@@ -98,8 +101,8 @@ export const checklistCertificadoOrigem = `
 • Dados do Adquirente
 • Acordo
 • Fatura Comercial
-• Valor das mercadorias
-• Quantidade - (trazer no formato quantidade x mercadoria)
+• Valor Total das mercadorias - Considerando todas as páginas faça a somatório do valor total informado por espécie de mercadoria usando a ferramenta calculator ou apenas recupere o valor total já informado no documento
+• Valor unitário de cada espécie de mercadoria - Considerando todas as páginas é o valor unitário multiplicado pela quantidade das mercadorias(quantidade X valor unitário, separando mercadorias diferentes por ';')
```

Comment:
De novo, apenas acrescentando um espaço além das vírgulas do @matheusmondaini-startse 
```suggestion
• Valor Total das mercadorias - Considerando todas as páginas, faça a somatório do valor total informado por espécie de mercadoria usando a ferramenta calculator ou apenas recupere o valor total já informado no documento
• Valor unitário de cada espécie de mercadoria - Considerando todas as páginas, é o valor unitário multiplicado pela quantidade das mercadorias (quantidade X valor unitário, separando mercadorias diferentes por ';')